dask/dataframe/utils.py

```diff
@@ -1,13 +1,21 @@
 from __future__ import absolute_import, division, print_function
 
 from collections import Iterator
+import datetime
 
 import numpy as np
 import pandas as pd
 import toolz
 
 from dask.async import get_sync
 
+# Pandas <0.17 doesn't have the datetime64+tz extension dtype
+try:
+    is_datetime64tz_dtype = pd.core.common.is_datetime64tz_dtype
```

Comment:
though generally I like explicit version checks for things like this (e.g. assume it exists >= a certain version). IOW, if this ever got renamed in pandas then this test would fail silently (rather than raising a nice error).